connectors/sources/confluence.py

```diff
@@ -1178,6 +1269,16 @@ async def _consumer(self):
             else:
                 yield item
 
+    def check_api_exceptions_and_raise(self):
+        failed_percentage = (
+            self.confluence_client.api_failed_count
+            / self.confluence_client.api_total_count
+        ) * 100
+        if failed_percentage >= API_FAILURE_THRESHOLD * 100:
+            msg = f"High percentage of API exceptions: {failed_percentage}%. Please review the logs for more information."
```

Comment:
Let's add the threshold to this error message too, and make it clear that this is calculated at the end of the sync.